mycore-webtools/src/main/java/org/mycore/webtools/properties/MCRPropertyHelper.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.webtools.properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.mycore.common.config.MCRComponent;
+import org.mycore.common.config.MCRConfigurationDir;
+import org.mycore.common.config.MCRConfigurationInputStream;
+import org.mycore.common.config.MCRRuntimeComponentDetector;
+import org.mycore.common.content.MCRContent;
+import org.mycore.common.content.MCRFileContent;
+import org.mycore.common.content.MCRURLContent;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+class MCRPropertyHelper {
+
+    private final LinkedHashMap<String, byte[]> configFileContents;
+
+    MCRPropertyHelper() throws IOException {
+        configFileContents = getConfigFileContents("mycore.properties");
+    }
+
+    public static LinkedHashMap<String, byte[]> getConfigFileContents(String filename) throws IOException {
```

Comment:
The methods `getConfigFileContents` and `getConfigFileStream` are almost exact duplicates of methods in `MCRConfigurationInputStream`. Why not modify and reuse the existing implementation? This duplication unnecessarily risks diversion when `MCRConfigurationInputStream` is modified in the future.

Reply:
I changed the existing api to return a LinkedHashMap<String, byte[]> to make it reusable.